examples/2d/bounding_2d.rs

```diff
@@ -224,13 +224,16 @@ fn setup(mut commands: Commands) {
             Vec2::new(40., 50.),
         )),
         Spin,
-        DesiredVolume::Aabb,
+        DesiredVolume::Circle,
```

Comment:
Why was this changed? I believe the example was intended to have AABBs and bounding circles alternating

![example](https://github.com/user-attachments/assets/da90a3bf-0271-4b52-85ec-34311de4de5a)